.github/workflows/actions/patch_image_and_check_diff/action.yml

```diff
@@ -123,6 +130,13 @@ runs:
         kubectl wait --for=condition=Ready pod --all -n ${{ inputs.sample-app-namespace }}
 
     # Log artifacts being used to verify whether the correct versions are being used
+    - name: Log CW Agent Operator spec container args before patching
+      id: default-cw-agent-operator-image
```

Comment:
Remove or update the id for this step too

Reply:
Fixed